pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -408,20 +418,35 @@ func (c *Controller) deleteNodeRoute(nodeName string) error {
 	return nil
 }
 
+func isIPsEqual(l0 []net.IP, l1 []net.IP) bool {
+	if len(l0) != len(l1) {
+		return false
+	}
+	l0Map := make(map[string]bool)
```

Comment:
In your case, there should be at most two IPs to compare? If that is the case, no need to create a map, but just iterate the slice to compare.

Reply:
Updated.